terraform/main.tf

```diff
@@ -88,3 +94,39 @@ module "aws_deploy-us-west-2" {
     aws = "aws.us-west-2"
   }
 }
+
+module "aws_deploy-dev1-eu-west-2" {
```

Comment:
I see correspondent UAT section has no env in the name so may cause confusion.

Reply:
Yes, it may need separate manual migration, thus not included here.